### PR TITLE
[README]: Installation step to run webapp

### PR DESCRIPTION
**Pull request by MachinisteWeb**
Hi @vberger!

I enhanced the README.md with all step I have due to follow to run the project.

I discover Rust with your project! Thx!

### CODE REVIEW

**Comment by elinorbgr** [comment]
I agree I should have put some installations instructions for to run it locally for development, thanks for taking the time to write this.

However, I don't think the large "Troubleshooting" part about how to fully install Rust on Windows would have its place in the README. Maybe just adding a small note between `2.` and `3.` about needing to install VS C++ and a link to the stackoverflow question would be enough?

**Comment by MachinisteWeb** [comment]
Good idea. It's done. Something else?

File: README.md [hunk 1]
```diff
@@ -7,3 +7,19 @@ A live version of the app is available here: https://vberger.github.io/Bayes-O-M
...
 the app is available in-app via the "Help" button, or as a
 [markdown file](https://github.com/vberger/Bayes-O-Matic/tree/master/static/help).
+
+
+
+## How to install
+
+### Prerequisites
+
+Before start installation, be sure the *C++ tools* is installed on your system. You can download it into the [Build Tools for Visual Studio 2019](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=16).
```
Comment (elinorbgr) [inline README.md]:
```suggestion
You will need to install the compiler for the Rust programming language. Do to so there are some prerequisites:

- On Windows, you need the *C++ tools* is installed on your system. You can download it into the [Build Tools for Visual Studio 2019](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=16).
- On Linux you'll need a working compilation toolchain. Your distribution likely provides the necessary package (for example `build-essential` on Debian).
- 
```

Comment (elinorbgr) [reply README.md]: I'd rather formulate it like so, to be more cross-platform. I don't know if there are prerequisites for macos though.

**Comment by MachinisteWeb** [comment]
@vberger, we can probably merge this one? Some change still need to be done?

**Comment by elinorbgr** [comment]
If you agree with my suggestion, maybe you could merge it into your branch, or give me the right to do so?

**Comment by MachinisteWeb** [comment]
I'm not sure to understand if you agree these modification. I suggest you to proceed following :

Your repository is used as « official » so, that means all PR any developer on the platform makes from his/her forked version should be reversed into your version when the PR is ready to merge.

Later, if any developer wants to have last modification from your repository, he/she can add a remote point on your repository with `git remote add` and do a `git fetch <name-of-remote-point>` on it to be up-to-date (but it is not something you need to worry).

So currently, the step are the following:

1. You accept pull request and merge my branch `Haeresis:doc-bruno-start-webapp` into your master `vberger:master` (with the button merge).
2. You are not able to close my branch because that remains on my repository. I can delete my branch `Haeresis:doc-bruno-start-webapp` myself (if I wish).
3. From your point of vue (your repository), all is OK. From mine and all others, I/We need to update `<other-repo>:master` from your new `vberger:master` to be up-to-date too (if I/We wish).

(If for some reason you decide to not apply this modification into your repository, you should have a « decline » or « close » button. So now, it is my responsibility or not to merge later my `Haeresis:doc-bruno-start-webapp` into `Haeresis:master` if I want).

Do not hesitate if something seems not clear :)

**Comment by elinorbgr** [comment]
I used the "change suggestion" feature of github to suggest a change to your PR, and given your emoji reaction, I suppose you agree with this change. So now this change needs to be applied to your branch before I merge it into my repo.

To do that, you can either [click on the appropriate button to accept my change](https://help.github.com/en/articles/incorporating-feedback-in-your-pull-request), or [check the checkbox to allow me to push commit into your PR branch](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork).

**Comment by MachinisteWeb** [comment]
Sorry, I have not see your change commit -_-'

My bad ! Thx a lot @vberger

**Comment by elinorbgr** [comment]
Okay, all is good. Thanks!

**Pull request merged**